### PR TITLE
Add/quivr api

### LINKED ISSUE #282: Adding QUIVR- Dump all your files and chat with it using your generative AI
https://github.com/QuivrHQ/quivr

i want to add this in the project

### PR DESCRIPTION
**Pull request by Maheshwari-Love**
closed #282

### CODE REVIEW

**Comment by Maheshwari-Love** [comment]
@ashitaprasad review this pr please

File: README.md [hunk 1]
```diff
@@ -59,8 +59,11 @@ You can start contributing by adding the following:
 | [Voyage AI](https://www.voyageai.com/) | [Link](https://docs.voyageai.com/reference/embeddings-api) | Y | Voyage AI provides API endpoints for embedding and reranking models. |
 | [AI/ML API](https://aimlapi.com/) | [Link](https://docs.aimlapi.com/) | Y |  An API aggregator that provides access to 100+ AI models via a single API. |
 | [Wit.ai](https://wit.ai/) | [Link](https://wit.ai/docs/http/20240304/) | Y | Wit.ai provides APIs to build natural language experiences. |
+| [Assembly API](https://www.assemblyai.com/) | [Link](https://www.assemblyai.com/docs/integrations) | Y | Built by AI experts, AssemblyAI’s Speech AI models include accurate speech-to-text for voice data (such as calls, virtual meetings, and podcasts), speaker detection. |
```
Comment (ashitaprasad) [inline README.md:62]: not a generative AI API

File: README.md [hunk 2]
```diff
@@ -59,8 +59,11 @@ You can start contributing by adding the following:
...
+| [QUIVR](https://www.quivr.com/) | [Link](https://www.quivr.com/) | Y | Turn private and enterprise knowledge into a personal AI assistant - learning and improving at every interaction. |
```
Comment (ashitaprasad) [inline README.md:63]: API docs not added.

File: README.md [hunk 3]
```diff
@@ -59,8 +59,11 @@ You can start contributing by adding the following:
...
 | [Clipdrop](https://clipdrop.co/) | [Link](https://clipdrop.co/apis/docs) | Y |  ClipDrop offers APIs for image upscaling, background removal, and other image enhancement features.|
 
+
```
Comment (ashitaprasad) [inline README.md:66]: Unnecessary empty line added.

**Pull request closed without merging**